.github/workflows/build-and-test.yml

```diff
@@ -48,20 +48,18 @@ jobs:
         CC: ${{ matrix.compiler }}
         LIBDQLITE_TRACE: 1
       run: |
-          for bin in unit-test integration-test \
-                     raft-core-fuzzy-test raft-core-integration-test \
-                     raft-core-unit-test raft-uv-integration-test \
-                     raft-uv-unit-test
-          do ./$bin || touch any-failed
-          done 2>&1 | tee -a test-suite.log
-          test '!' -e any-failed
+          tests="integration-test \
```

Comment:
Can you add a TODO somewhere to revert this once we figure out why it hangs?

Reply:
Done